content/docs/user-guide/project-config/cli-reference.md

```diff
@@ -1,7 +1,1704 @@
+
+- **`-ohf OVERRIDE_HOME_FOLDER, --override-home-folder OVERRIDE_HOME_FOLDER`**
+
+  Overrides the default home folder with the specified folder. By default, the home folder is the user folder. 
+
+
+<!-------------------------------------------------------------->
+
+## `edit-engine-properties`
+
+Edits the specified engine's properties by modifying the `engine.json` file. 
+
+### Format
+
+```cmd
+edit-engine-properties [-h] (-ep ENGINE_PATH | -en ENGINE_NAME)
+                        [-enn ENGINE_NEW_NAME]
+                        [-ev ENGINE_VERSION]
+```
+
+### Usage
+
+Updates the `engine_name` field in the `engine.json` file located at the specified engine's folder. 
+
+```cmd
+o3de.bat edit-engine-properties --engine-path ENGINE_PATH --engine-new-name ENGINE_NEW_NAME
+
+// or
+
+o3de.bat edit-engine-properties --engine-name ENGINE_NAME --engine-new-name ENGINE_NEW_NAME
+```
+
+
+### Optional parameters
+
+  
+- **`-h, --help`**
+
+  Shows the help message.
+
+- **`-ep ENGINE_PATH, --engine-path ENGINE_PATH`**
+
+  The path to the engine.
+
+- **`-en ENGINE_NAME, --engine-name ENGINE_NAME`**
+
+  The name of the engine.
+
+
+**Engine properties:**
+
+The following parameters modify the specified engine's properties.
+  
+- **`-enn ENGINE_NEW_NAME, --engine-new-name ENGINE_NEW_NAME`**
+
+  Sets the engine's name. 
+
+- **`-ev ENGINE_VERSION, --engine-version ENGINE_VERSION`**
+
+  Sets the engine's version.
+
+
+<!-------------------------------------------------------------->
+
+## `edit-project-properties`
+
+Edits the specified project's properties by modifying the `project.json` file. 
+
+### Format 
+
+```cmd
+edit-project-properties [-h]
+                        (-pp PROJECT_PATH | -pn PROJECT_NAME)
+                        [-pnn PROJECT_NEW_NAME]
+                        [-po PROJECT_ORIGIN]
+                        [-pd PROJECT_DISPLAY]
+                        [-ps PROJECT_SUMMARY]
+                        [-pi PROJECT_ICON]
+                        [-at [ADD_TAGS [ADD_TAGS ...]] | -dt
+                        [DELETE_TAGS [DELETE_TAGS ...]] | -rt
+                        [REPLACE_TAGS [REPLACE_TAGS ...]]]
+```
+
+### Usage
+
+Updates the "project_name" field within the gem.json located at the supplied project path or at the path of the registered project with <project-name>
+
+```cmd
+o3de.bat edit-project-properties --project-path PROJECT_PATH --project-new-name PROJECT_NEW_NAME
+o3de.bat edit-project-properties --project-name PROJECT_NAME --project-new-name PROJECT_NEW_NAME
+```
+
+### Optional parameters
+
+- **`-h, --help`**
+
+  Shows the help message.
+
+- **`-pp PROJECT_PATH, --project-path PROJECT_PATH`**
+
+  The path to the project.
+
+- **`-pn PROJECT_NAME, --project-name PROJECT_NAME`**
+
+  The name of the project.
+
+- **`-at [ADD_TAGS [ADD_TAGS ...]], --add-tags [ADD_TAGS [ADD_TAGS ...]]`**
+
+  Adds tag(s) to the `user_tags` property. To add multiple tags, use a space delimited
+list (for example, `-at A B C`).
```

Comment:
```suggestion
Adds tags to the `user_tags` property. To add multiple tags, use a space-delimited list. For example: `-at A B C`.
```
* The reader will understand that "tags" can mean "one or more" without the parenthesis around 's'.
* Hyphenated "space delimited", as it's a compound adjective describing "list".
* Reformatted the example at the end for consistency with previous examples in this CLI ref.